packages/expo-cli/src/commands/start/TerminalUI.js

```diff
@@ -200,9 +200,9 @@ export const startAsync = async (projectDir, options) => {
 
               let sent = false;
               try {
-                await Exp.sendAsync(sendTo, lanAddress);
+                const result = await Exp.sendAsync(sendTo, lanAddress);
+                sent = result.sent;
```

Comment:
looks like we're not regressing anything here but sent/result seem unused :O

Reply:
This is the only place result.sent is being used. And I forced it. 
I am going to just delete line 204 and remove result.sent from the server response as well.

`Sent link successfully` is gated by sendAsync throwing an error if it's not. (sent is hardcoded 'true' on the server side).